CHANGELOG.md

```diff
@@ -4,7 +4,11 @@
 
 ### New features
 
-* [#60](https://github.com/nrepl/nrepl/issues/60): Implemented EDN transport.
+* [#60](https://github.com/nrepl/nrepl/issues/60): Implemented EDN transport
+
+### Changes
+
+* [#137](https://github.com/nrepl/nrepl/pull/137): Expanded Encode transport to allow more key types
```

Comment:
I think you meant to say Bencode here. :-)

Reply:
I think you can also mention that the key types are symbol and keywords here.

Probably you'd be a good idea to add a couple of examples about the conversation of requests from external to internal representation to the bencode and edn sections of the manual.

Reply:
Will correct. I've also been working on internal v.s. external representations. Would like to jot down some notes (as an issue, probably) to discuss before getting to documenting.

Reply:
👍

Reply:
Fixed the changelog